src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1220,6 +1220,8 @@ private static bool IsDynamicStatics(TypeDesc type)
             }
             else
             {
+                // At this point, we knew it is a virtual call to targetMethod method, if it is a default interface method call, it should go through instantiating stub
+                useInstantiatingStub = useInstantiatingStub || (targetMethod.OwningType.IsInterface && !originalMethod.IsAbstract);
```

Comment:
@cshung - thanks for tracking this down.
I think this comment could use a bit of rewording. Does the "if it is" apply to the first part of the sentence or the last? Perhaps "In addition to the conditions handled above, we also need an instantiating stub if it is a default interface method". Does that capture it?